mod/host.py

```diff
@@ -1152,7 +1152,10 @@ def report_current_state(self, websocket):
             for symbol, value in pluginData['outputs'].items():
                 if value is None:
                     continue
-                websocket.write_message("output_set %s %s %f" % (pluginData['instance'], symbol, value))
+                if isinstance (value, float):
+                    websocket.write_message("output_set %s %s %f" % (pluginData['instance'], symbol, value))
+                if isinstance (value, str):
```

Comment:
Why save the message at all?
And it is currently reached via  `desktop.pedalboard.pedalboard("setOutputPortValue", ...)